packages/ra-no-code/src/Admin.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Admin as RaAdmin, Resource } from 'react-admin';
+import localStorageDataProvider from 'ra-data-local-storage';
+import { CreateBuilder, EditBuilder, ListBuilder } from './builders';
+import {
+    useResourceConfigurations,
+    ResourceConfigurationProvider,
+} from './ResourceConfiguration';
+import { Layout, Ready } from './ui';
+
+const dataProvider = localStorageDataProvider();
+
+export const Admin = () => (
+    <ResourceConfigurationProvider>
+        <InnerAdmin />
+    </ResourceConfigurationProvider>
+);
+
+const InnerAdmin = () => {
+    const { resources } = useResourceConfigurations();
```

Comment:
The final 's' in the hook name feels weird. Did you mean `useResourcesConfiguration`?